test/gtest/ucp/test_ucp_wakeup.cc

```diff
@@ -231,9 +231,9 @@ void test_ucp_wakeup_events::do_tx_rx_events_test(unsigned events)
     if (events & UCP_WAKEUP_RX_AM) {
         /* UCP_WAKEUP_RX_AM may cause events on sender side because of internal
          * messages arriving, these are not TX-events */
-        EXPECT_LE(nfd_exp, poll(polled, 2, 200*ucs::test_time_multiplier()));
+        EXPECT_LE(nfd_exp, poll(polled, 2, 10000));
     } else {
-        EXPECT_EQ(nfd_exp, poll(polled, 2, 200*ucs::test_time_multiplier()));
+        EXPECT_EQ(nfd_exp, poll(polled, 2, 10000));
```

Comment:
I would propose to use `ucs::test_time_multiplier()` because this function make difference between regular and run under valgrind.
For example `1000 * ucs::test_time_multiplier()`